travis/build-and-test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -e
+
+build_order=(opm-common opm-parser opm-material opm-core opm-output opm-grid opm-simulators ewoms opm-upscaling)
+
+# This shell script should be started with the name of a module as
+# only only command line argument. It will start by building all
+# upstream modules, then it will build and test the module of interest
+# and all downstream modules.
+#
+# Before invoking this script all the modules should have been cloned
+# in sibling directories, so that this script will see this directory
+# structure:
+#
+#   opm-common/
+#   opm-parser/
+#   opm-material/
+#   opm-output/
+#   opm-core/
+#   opm-grid/
+#   opm-simulators/
```

Comment:
You are creating extra work for yourself with all this documentation :p

Now you should also see `opm-upscaling` and `ewoms`.